services/src/main/scala/cromwell/services/cost/CostCatalogService.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.services.cost
+
+import akka.actor.{Actor, ActorRef}
+import com.google.cloud.billing.v1._
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import common.util.StringUtil.EnhancedToStringable
+import cromwell.services.ServiceRegistryActor.ServiceRegistryMessage
+import cromwell.util.GracefulShutdownHelper.ShutdownCommand
+
+import java.io.{FileOutputStream, ObjectOutputStream}
+import java.time.{Duration, LocalDateTime}
+import scala.jdk.CollectionConverters.IterableHasAsScala
+import java.time.temporal.ChronoUnit.HOURS
+case class CostCatalogMessage() extends ServiceRegistryMessage {
+  override val serviceName = "CostCatalogService"
+}
+
+case class ExpiringCostCatalog(catalog: Map[CostCatalogKey, CostCatalogValue], fetchTime: LocalDateTime)
```

Comment:
Given that this is a GCP-specific component in a generic package, can we sprinkle some `GCP` into the naming?